components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/internal/processor/JsonRestProcessor.java

```diff
@@ -104,6 +108,11 @@ protected void processRequest(Exchange exchange) throws SalesforceException {
             });
             break;
 
+        case RECENT:
+            // handle known response type
```

Comment:
Ah okay there are others doing this, maybe you can change the code to do like they do so its similar?